### PR TITLE
Feature/cl updates late jan

### PR DESCRIPTION
**Pull request by lowriech**

### CODE REVIEW

**Comment by lowriech** [comment]
Yes, there are probably a thousand console.logs to clean up :)

File: src/hooks/layers/useLayers.js [hunk 1]
```diff
@@ -55,12 +56,13 @@ export function useLayers(
...
     if (mapLoaded) {
       if (layerGroup === LayerName.RiskReduction) {
+        clearTimeout(viewportLockTimeout.current);
         map.setMaxPitch(75);
         map.dragRotate.enable();
       } else {
         map.setPitch(0, { duration: 2000 });
-        map.rotateTo(0, { duration: 2000 });
-        setTimeout(() => {
+        map.rotateTo(0, { animationDu: 2000 });
```
Comment (gislawill) [inline src/hooks/layers/useLayers.js:64]: is `animationDu` the right key here? Checking to make sure it's not a typo

**Review by gislawill — APPROVED** [review]
Lots of good stuff in here @lowriech — please merge if it all good to you 🫐

**Comment by lowriech** [comment]
Consolidated some work, and redeployed to staging.

**Pull request merged**